sdk/core/azure-core/CHANGELOG.md

```diff
@@ -0,0 +1,22 @@
+# Change Log azure-core
+
+## Version 1.1.0 (2019-11-26)
+- Added support for creating reactor-netty-http client from an existing client.
+- Added UserAgent helper methods for fetching client name and version from pom file.
+- Added toReactorContext to FluxUtil.
+- Logging exception at warning level, and append stack trace if log level is verbose.
+- Fixed HttpLoggingPolicy to take null HttpLogOptions.
+- Changed the User agent format.
+- Hided the secrets from evnironment variable.
+- UserAgentPolicy is using the value stored in the policy no matter what is stored in the passed request. Also, removed the service version from User agent format.
+- Added Iterable<T> overload for IterableStream<T>.
+- Reduce Prefetch Limit for PagedIterable and IterableStream.
+- Ensure HTTPS is used when authenticating with tokens.
+
```

Comment:
Add Following as well
- New AddHeadersFromContextPolicy to override headers in HttpRequest
-  Override header name in RequestIDPolicy

Reply:
Is this 1.1.0? My updates are for 1.1.0 release